apps/web/src/pages/templates/TemplatesDigestPlaygroundPage.tsx

```diff
@@ -0,0 +1,55 @@
+import { Stack } from '@mantine/core';
+import styled from '@emotion/styled';
+import { useNavigate, useParams } from 'react-router-dom';
+
+import PageContainer from '../../components/layout/components/PageContainer';
+import { ArrowButton, colors, Title, Text, Button } from '../../design-system';
+import { parseUrl } from '../../utils/routeUtils';
+import { ROUTES } from '../../constants/routes.enum';
+
+const Heading = styled(Title)`
+  color: ${colors.B40};
+  font-size: 40px;
+`;
+
+const SubHeading = styled(Text)`
+  font-size: 20px;
+`;
+
+const DigestWorkflowHolder = styled.div`
+  width: 400px;
+  height: 400px;
+  background: ${colors.vertical};
+`;
+
+const LinkStyled = styled.a`
+  font-size: 16px;
+  color: #2b85df;
+  text-decoration: underline;
+`;
+
+export const TemplatesDigestPlaygroundPage = () => {
+  const { templateId = '' } = useParams<{ templateId: string }>();
+  const navigate = useNavigate();
+
+  return (
+    <PageContainer title="Digest Workflow Playground" style={{ padding: '32px' }}>
+      <ArrowButton label="Go Back" onClick={() => navigate(-1)} />
```

Comment:
How does `navigate(-1)` reacts if someone loads the URL directly in the navigator? I am afraid the go back button wouldn't work.

Reply:
it works and navigates to the `/templates` page